x-pack/plugin/eql/qa/common/src/main/resources/test_queries.toml

```diff
@@ -391,8 +691,12 @@ sequence
   [process where opcode == 2] by process_path
   [file where event_subtype_full == "file_delete_event"] by file_path
 | head 4
-| tail 2'''
-expected_event_ids  = [67, 68, 69, 70, 72, 73, 74, 75]
+| tail 2
+'''
+expected_event_ids  = [67, 68,
+                       69, 70,
+                       72, 73,
+                       74, 75]
```

Comment:
These are incorrectly formatted (as is the rest of the group under it) - the sequence has 4 items not 2 on a line.